packages/financial-templates-lib/src/logger/SlackTransport.ts

```diff
@@ -52,7 +52,7 @@ export const SLACK_MAX_CHAR_LIMIT = 3000;
 // Note: info is any because it comes directly from winston.
 function slackFormatter(info: any): SlackFormatterResponse {
   try {
-    if (!("level" in info) || !("at" in info) || !("message" in info))
+    if (!("level" in info) || !("at" in info) || !("message" in info) || !("sessionId" in info))
```

Comment:
Do fields in the detaultMeta get injected into info? Just wanted to double check.